### PR TITLE
Add php7-ldap

### PR DESCRIPTION
**Pull request by shamsway**
Added necessary requirements to Dockerfile to support LDAP authentication in LibreNMS

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,8 +28,10 @@ RUN apk --update --no-cache add \
     git \
     graphviz \
     imagemagick \
+    libldap \
```
Comment (crazy-max) [inline Dockerfile]: Not required since `php7-ldap` depends of

File: Dockerfile [hunk 2]
```diff
@@ -28,8 +28,10 @@ RUN apk --update --no-cache add \
...
     monitoring-plugins \
     mtr \
+    musl \
```
Comment (crazy-max) [inline Dockerfile]: Not required since `php7-ldap` depends of

File: Dockerfile (hunk 1 above)
Comment (shamsway) [reply Dockerfile]: Removed

File: Dockerfile (hunk 2 above)
Comment (shamsway) [reply Dockerfile]: Removed

**Comment by shamsway** [comment]
I have made the requested changes

**Comment by crazy-max** [comment]
LGTM thanks :+1:

**Pull request merged**